assets/scss/components/_content-item.scss

```
@@ -317,6 +265,72 @@ $content-reaction-offset: -14px; // -30px to not offset the meta
     }
 }
 
+.write-tutorial {
+    display: flex;
+    align-items: center;
+
+    margin: 0 1rem;
+
+    &:not(:last-child) {
+        margin-bottom: 2rem;
+    }
+
+    width: 100%;
+    height: $content-item-height;
+
+    background-color: $color-primary;
+
+    border-radius: 8px;
+
+    color: $primary-100;
+
+    @include shadow-2;
+    transition: box-shadow $transition-duration ease-in-out;
+
+    &:hover {
+        @include shadow-3;
+
+        .btn-write-tutorial {
+            background-color: $true-white;
+        }
+    }
+
+    .write-tutorial-text {
+        flex: 1;
+        text-align: center;
+
+        p {
+            margin: 0;
+            font-size: $size-9;
+
+            &.lead {
+                font-size: $size-8;
+                font-weight: bold;
+            }
+        }
+    }
+
+    .btn-write-tutorial {
+        padding: .4rem 2rem;
+        margin-right: 2.5rem;
+
+        border-radius: 8px;
+
+        background-color: $primary-100;
+        color: $primary-800;
+
+        font-size: $size-8;
+        user-select: none;
+
+        transition:            color $transition-duration ease-in-out,
+                    background-color $transition-duration ease-in-out;
```

Je trouve personnellement cette indentation horrible ! :)
```suggestion
        transition: color $transition-duration ease-in-out,
                    background-color $transition-duration ease-in-out;
```

---

C'est pas aligné du coup :(

---

Bah si c'est aligné par rapport à `transition:` ! D'ailleurs tu l'alignes aussi comme ça ailleurs dans le code. :)

---

Ah possible alors :D En réalité je n'y suis pas à ce point attaché, donc je peux changer sans souci. C'est potentiellement une mise en forme automatique d'ailleurs, je ne me souviens plus de l'avoir aligné ainsi.